queue_services/entity-filer/src/entity_filer/filing_processors/__init__.py

```diff
@@ -41,7 +41,7 @@ def create_address(address_info: Dict, address_type: str) -> Address:
                       country=pycountry.countries.search_fuzzy(address_info.get('addressCountry'))[0].alpha_2,
                       postal_code=address_info.get('postalCode'),
                       delivery_instructions=address_info.get('deliveryInstructions'),
-                      address_type=address_type
+                      address_type=address_type.replace('Address', '')
```

Comment:
Would it be more clear to have a switch or if statements, kinda like:

if (address_type == "addressMailing") then
  address_type = "mailing"
else if (address_type == "addressDelivery") then
  address_type = "delivery"

Reply:
Also, is case correct?

Reply:
Also, can this be unit tested?

Reply:
1) yes, I thought about it, but it sort of defeats the purpose of iterating over dictionary keys in the caller...but I think I will change it since you've mentioned it.

2) This is how it comes in from the filing, and the schema should enforce that, but it doesn't hurt to make it case-insensitive

3) It shouldn't need a new test case as we already test IA using this format

Reply:
1. or add a comment explaining why / what you're doing

Reply:
2. I thought it was "addressMailing"?

Reply:
3. ok

Reply:
1) done, 2) no, it was mailingAddress, 3) good